syntax="proto3";
 
message UserName {
    message FullName { 
        string first_name = 1;
        string last_name = 2;
        string second_name = 3;
    }

    FullName name = 1;
}

message UserAddr {
    string addr = 1;
}